storages/backends/gcloud.py

```diff
@@ -168,10 +178,14 @@ def _save(self, name, content):
         if CONTENT_TYPE not in blob_params:
             upload_params[CONTENT_TYPE] = file_object.mime_type
 
+        if self.gzip and upload_params.get(CONTENT_TYPE) in self.gzip_content_types:
+            content = self._compress_content(content)
+            file_object.blob.content_encoding = 'gzip'
```

Comment:
Alternatively I would just write

```python
blob_params['content_encoding'] = 'gzip'
```

since it ends up being set just below.

Reply:
Yeah, good one. Thanks!